.readthedocs.yaml

```diff
@@ -0,0 +1,36 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
+    # You can also specify other tool versions:
+    # nodejs: "16"
+    # rust: "1.55"
+    # golang: "1.17"
+  jobs:
+    pre_install:
+      - pip install sphinxcontrib-apidoc
+      - pip install --upgrade m2r2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   builder: html
+   configuration: docs/source/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
```

Comment:
Let's remove all the commented out code. It's just confusing for future readers

Reply:
Commented code removed